### DISCUSSION TITLE
Data not being added to DB

### DISCUSSION THREAD

**Discussion by studiopatagon** [jeremyevans/roda#314] [General]
I have an app using Sequel that has a very simple model/table but for some reason the data isn't being saved. No error is being thrown. Is there a way to debug to see if the application is even trying to add data to the database?

**Comment by adam12** [comment]
Is this still relevant? I'm not sure I've ever seen a Closed icon on a discussion before.

I'd check a few things.

1. Sequel is configured to complain loudly upon save failures. Ensure you're not skipping this. [`raise_on_save_failure`](https://sequel.jeremyevans.net/rdoc/classes/Sequel/Model/ClassMethods.html#attribute-i-raise_on_save_failure).

2. Assign a logger to Sequel `DB` and see if it's indeed being rolled back. `DB.logger = Logger.new($stdout)`.

3. Break just before `save` and see if it's valid or has errors.

**Reply by studiopatagon** [reply]
Yeah I figured it out... I don't see a way to 'delete' discussions so I closed it 😑

Thanks for the logger advice. It's what I had been looking for.
